examples/ray-joblib.py

```diff
@@ -0,0 +1,66 @@
+"""
+Optuna example that optimizes a classifier configuration for the Iris dataset using Ray with
+joblib backend.
+
+In this example, we optimize a classifier configuration for Iris dataset. Classifiers are from
+scikit-learn. We optimize both the choice of classifier (among SVC and RandomForest) and their
+hyper parameters.
+
+"""
+import logging
+
+import joblib
+import ray
+from ray.util.joblib import register_ray
+import sklearn.datasets
+import sklearn.ensemble
+import sklearn.model_selection
+import sklearn.svm
+
+import optuna
+
+
+# Get a URL of the Ray dashboard.
+ray.init()
```

Comment:
You can use here
```suggestion
ray.init(address=“auto”)
```
If this is running on a cluster and you do not explicitly set the RAY_ADDRESS=auto env variable.

Reply:
If there is no Ray redis address,  this command fails due to "ConnectionError" as follows.
<img width="895" alt="スクリーンショット 2021-02-03 11 15 51" src="https://user-images.githubusercontent.com/38826298/106688840-3f289200-6612-11eb-81ef-62c9977c3625.png">

So, I fix the following way. Do you have any concerns?
```python
try:
    ray.init(address="auto")
except ConnectionError:
    ray.init()
```

Reply:
This is relevant after you have started a cluster with ‘ray up’ for example, you will be able to run on multiple nodes. This will not crash on a cluster (will not work on local machine)